x/mint/types/minter_test.go

```diff
@@ -1,60 +0,0 @@
-package types
-
-import (
-	"testing"
-
-	"github.com/stretchr/testify/require"
-
-	"cosmossdk.io/math"
-)
-
-func TestNextInflationRate(t *testing.T) {
```

Comment:
uhhhhhhhhhhh are we not going to have tests ????

Reply:
logic was moved to `abci.go` and it's being tested in `module_test.go`